bot/seasons/evergreen/bookmark.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import random
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours, ERROR_REPLIES
+
+log = logging.getLogger(__name__)
+
+
+class Bookmark(commands.Cog):
+    """A cog that creates personal bookmarks by relaying a message to the user's DMs."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="bookmark", aliases=("bm", "pin"))
+    async def bookmark(self, ctx: commands.Context, target_message: discord.Message, *,
+                       title: str = "Bookmark") -> None:
+        """Send you a link to the provided message in DM."""
+        log.info(f"{ctx.author} bookmarked {target_message.jump_url} with hints {title}.")
+
+        if not title:
+            title = "No hint provided."

```

Comment:
You use the term "title" as the name of the parameter, which hints at the provided string being used as the title of the bookmark. I'm not really sure what "hint" means in this context, but it comes out a bit out of nowhere for me. Why not stick to the term "title" to make the user experience more consistent?

In addition, instead of adding some generic message, you could also just remove the title field completely if no title was set. That reduces the size of the embed and a generic "no title" message does not add any relevant information to the bookmark anyway.

Reply:
Removing the title itself is not possible(for me ) , also it will break the symmetry , will stick to `No title provided`

Reply:
> Removing the title itself is not possible(for me )

Why is it not possible? If you don't know how to, we could help you with that.

> also it will break the symmetry , will stick to No title provided

What symmetry?